packages/xrpl/src/models/utils/flags.ts

```diff
@@ -20,7 +21,7 @@ import {
   PaymentChannelClaimFlagsInterface,
   PaymentChannelClaimFlags,
 } from '../transactions/paymentChannelClaim'
-import type { Transaction } from '../transactions/transaction'
+import { type Transaction } from '../transactions/transaction'
```

Comment:
Why was this changed? Shouldn't this get around the cycle issue?

Reply:
This fixed a different cycle that was happening between Flags and Transaction (there were multiple :P)

Reply:
Nevermind - reverting this fixed the circular import - nice catch!